openedx/features/edly/utils.py

```diff
@@ -261,3 +265,28 @@ def send_comments_reply_email_to_comment_owner(comment, context):
     if parent_comment.user_id != comment.thread.user_id and parent_comment.user_id != comment.user_id:
         recipients = [parent_comment.user_id]
         send_bulk_mail_to_students.delay(recipients, message_context, 'comment_reply')
+
+
+def send_contact_us_email(data):
+    recipient_list = settings.CONTACT_US_SUPPORT_EMAILS
+    context = {
+        'requester_name': data["requester"]["name"],
+        'requester_email': data["requester"]["email"],
+        'requester_phone_number':  data["custom_fields"][1]['phone_number'],
```

Comment:
nit: be consistent with single quotes.

Single quotes are easy to put plus since the machine has to read the program we normally use single code while writing the code and double quotes are used when we are using the mention's in the docstrings or comment for human's to read.